# This patch is from Al Reay in bug 842041. It is based on the discussion at:
# http://www.ogre3d.org/forums/viewtopic.php?f=2&t=70052

--- OgreMain/src/OgreSIMDHelper.h	2012-07-03 21:01:01.831381577 +1200
+++ OgreMain/src/OgreSIMDHelper.h	2012-07-03 21:01:32.506998075 +1200
@@ -48,7 +48,7 @@
 // thing. The size of the allocated block seems to be irrelevant.
 #define __OGRE_SIMD_ALIGN_STACK()   _alloca(16)
 
-#elif OGRE_CPU == OGRE_CPU_X86 && (OGRE_COMPILER == OGRE_COMPILER_GNUC || OGRE_COMPILER == OGRE_COMPILER_CLANG)
+#elif OGRE_CPU == OGRE_CPU_X86 && OGRE_COMPILER == OGRE_COMPILER_CLANG
 //
 // Horrible hack to align the stack to a 16-bytes boundary for gcc.
 //
